### PR TITLE
Added Cart page and corrected the wishlist page

### PR DESCRIPTION
**Pull request by uday-nv-work**
Added Cart Page and Linked Wishlist page correctly

### CODE REVIEW

File: components/Wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.cart-items {
+	width: 100%;
+	padding-bottom: 3rem;
+}
+.cart-header {
+	text-align: center;
+	width: inherit;
+	padding: 25px;
+}
```
Comment (Shubhamkr2610) [inline components/Wishlist/wishlist.css]: suggestion: you can use rem for more than 20 px

**Review by rutu14 — APPROVED** [review]
The Image on cart page seems to not scale down and takes height of the container. 
Everything else looks good.

**Review by Shubhamkr2610 — APPROVED** [review]
Code is clean . Nice work . Everything looks good to me . You can merge it .

File: components/Cart/cart.html [hunk 2]
```diff
@@ -41,15 +41,15 @@
 					</div>
 					<div class="badge-wrapper"
 						><li class="list-style">
-							<a class="link" href="#">
+							<a class="link" href="/components/Wishlist/wishlist.html">
 								<i class="fa fa-heart nav-icons"></i>
 								<div class="badge icon">5</div>
 							</a>
```
Comment (sudipkundu999) [inline components/Cart/cart.html:44]: badge
The text is barely readable here, check the contrast ratio.
Try using a dark background color if using a light text color and vice-versa.

File: components/Wishlist/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+				<div class="nav-brand"
+					><img class="logo" src="/assets/logo.svg" alt="logo" />
+					<a class="component-library-link nav-icons" href="../../index.html">
+						<i class="fa">Xero</i></a
+					></div
+				>
+				<div class="search-bar">
+					<input
+						class="input"
+						placeholder="Search for products, brands and more..."
```
Comment (sudipkundu999) [inline components/Wishlist/wishlist.html]: the placeholder is not totally visible, either increase the width or shorten the text of the placeholder.
placeholder

**Review by sudipkundu999 — CHANGES_REQUESTED** [review]
UI and code look great. Good work.
Just fix the 2 minor UI issues.

**Comment by rishabh486** [comment]
Screenshot (48)
logo image is not loading other than this good to merge

**Review by sudipkundu999 — APPROVED** [review]
Good to merge!

**Pull request merged**